packages/sections/src/variant/InSilicoPredictors/Body.tsx

```diff
@@ -37,6 +39,11 @@ const columns = [
     label: "Score",
     renderCell: ({ score }) => score ?? naLabel,
   },
+  {
```

Comment:
"Normalised score" for label

Reply:
No no, the score we are using to set the position of the dots is "NormalisedScore" but the one we want to display is the method `score`. This is why are different. This was already review with Annalisa and David

Reply:
Though possibly plot tooltip should have normalised score and raw score?